templates/vue/src/components/SettingsModal.vue

```diff
@@ -35,6 +35,15 @@
               {{ autoLayout ? "Enabled" : "Disabled" }}
             </b-form-checkbox>
           </b-form-group>
+          <b-form-group
+            label="Superusers Override Permissions"
+            description="When this is enabled, all WordPress superusers (Administator roles, Editor roles or the
```

Comment:
Can you change this to the following:
`If enabled, editors of this tapestry would be able to see and edit all nodes even for nodes with the "read" permission off. If disabled, superusers will be able to use the filter to view such nodes, but they won't appear in the tapestry by default. Note: Editors of this tapestry include users with the Administator or Edit roles, and the author of this Tapestry.`